agent/agent-tooling/src/main/java/io/opentelemetry/sdk/metrics/internal/view/MetricView.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.opentelemetry.sdk.metrics.internal.view;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import java.util.HashSet;
+import java.util.Set;
+
+@SuppressWarnings("rawtypes")
+enum MetricView {
+  HTTP_CLIENT_VIEW("http.client.duration", httpClientDurationAttributeKeys(), false),
+  HTTP_SERVER_VIEW("http.server.duration", httpServerDurationAttributeKeys(), true),
+  RPC_CLIENT_VIEW("rpc.client.duration", rpcClientDurationAttributeKeys(), false),
+  RPC_SERVER_VIEW("rpc.server.duration", rpcServerDurationAttributeKeys(), false);
```

Comment:
what about synthetic for rpc server?

Reply:
I realized that rpc instrumentation doesn't capture http.user_agent